docs/dev/how-to-debug-minion-import-and-export-tasks.md

```diff
@@ -0,0 +1,102 @@
+# How to debug Minion import and export tasks
+
+For imports on the pro platform, and exports from the pro platform and imports on the public platform, we use Minion tasks.
+
+Those tasks are run by a Minion worker service, which is configured in /etc/systemd/system/minion\@off.service
+
+## Log files
+
+in the logs directory, we have:
+- minion.log - states which tasks are started
+- minion_log4perl.log - debug messages from the Perl code that implements the Minion tasks
+
+## Checking the status of Minion tasks (jobs)
+
+````
+off@off:/srv/off$ (off) ./scripts/minion_producers.pl minion job 
+[..]
+132132  inactive  pro.openfoodfacts.org  update_export_status_for_csv_file
+132131  failed    openfoodfacts.org      import_csv_file
+132130  finished  pro.openfoodfacts.org  export_csv_file
+````
+
+## Debugging failed import of pro platform data on public platform
+
+The following are notes on how a specific issue was debugged in production:
+
+We now are loading automatically Systeme U (a big French retailer) imports in the producers platform.
+
+The checkbox in the Systeme U organization (admin field) to automatically export new products is checked, but the products have not been loaded on the producers platform.
+
+We can launch the export and import manually:
+
+off@off-pro:/srv/off-pro$ (off-pro) scripts/export_and_import_to_public_database.pl --query states_tags=en:to-be-exported --owner org-systeme-u
+
+And we can see the details of the corresponding minion jobs:
+
+````
+off@off-pro:/srv/off-pro$ (off-pro) ./scripts/minion_producers.pl minion job
+[..]
+132132  inactive  pro.openfoodfacts.org  update_export_status_for_csv_file
+132131  failed    openfoodfacts.org      import_csv_file
+132130  finished  pro.openfoodfacts.org  export_csv_file
+````
+
+In the off container, there is little useful information in the logs:
+
+/srv/off/logs/minion.log:
+
+````
```

Comment:
Note you should normally use 3 backtick 
the 4 backtick is reserved for the case where you must include 3 backtick inside a code block

Reply:
Thanks